src/components/Profile/VpDelegationBox.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+
+import { AsyncStateResultState } from 'decentraland-gatsby/dist/hooks/useAsyncState'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Container } from 'decentraland-ui/dist/components/Container/Container'
+import Grid from 'semantic-ui-react/dist/commonjs/collections/Grid/Grid'
+
+import { DelegationResult } from '../../clients/SnapshotGraphqlTypes'
+import Empty from '../Common/Empty'
+import SkeletonBars from '../Common/SkeletonBars'
+import DelegatorCardProfile from '../Delegation/DelegatorCardProfile'
+import Scale from '../Icon/Scale'
+
+import { ProfileBox } from './ProfileBox'
+import './VpDelegationBox.css'
+
+interface Props {
+  delegation: DelegationResult
+  delegationState: AsyncStateResultState<DelegationResult, DelegationResult>
```

Comment:
If you are only using loading I'd simplify this to `isLoadingDelegations: boolean`